corehq/apps/app_manager/tests/test_create_child_module_command.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+from django.core.management import call_command
+
+
+from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+
+
+class CreateShadowChildrenTest(TestCase):
```

Comment:
I just removed the management command since with the "version" support I added it is no longer needed. Instead, I moved these tests to the corresponding function instead.